### ISSUE TITLE
[FR] Run appflowy on Heroku

### ISSUE THREAD

**Issue by lays147** [AppFlowy-IO/appflowy#300]
**1~3 main use cases of the proposed feature**
It would be nice to have a setup to run appflowy on Heroku. I'm trying to do that by using the Docker image, but no luck. On the web exists tutorials about pushing rust and flutter to heroku, but not both together, but I'm sure that could be a way. I may try, but for the ones that know the codebase could be easier? :smiley: 

**what types of users can benefit from using your proposed feature**
Everyone that want to give a try to AppFlowy.

**Additional context**

**Comment by Driesvanherpe** [comment]
I'd like to take this on if possible

**Comment by appflowy** [comment]
Sure, thank you for giving it a shot. If you have quick questions, please reach out to me on Discord.

**Comment by pftg** [comment]
Yep, with one button deploy everybody would be able to run own instance in no time and no much effort!

**Comment by annieappflowy** [comment]
@Driesvanherpe hello, any update on this?

**Comment by Poly-Pixel** [comment]
I might be misunderstanding this, but AppFlowy is not a web application yet - right now, it is only for desktop. Therefore, I don't know how you would deploy it to Heroku, which to my understanding, is a platform for web apps.

**Comment by aarhusgregersen** [comment]
Chiming in here to ditto what @Poly-Pixel says. I initially thought it was a web-based app too, but this currently only builds for desktop, so that's not the case.

When a web implementation becomes common, we should make it docker-based ASAP.

**Comment by vip20** [comment]
I might be wrong, why not use firebase?

**Comment by annieappflowy** [comment]
for heroku, we probably need a web version of AppFlowy the platform does not run native app. I think you could possibly deploy AppFlowy Cloud there

**Comment by thuvh** [comment]
should use render instead heroku?

**Comment by almereyda** [comment]
The AppFlowy web app is available now. One would have to decompose the existing Cloud distribution into similar manifests for Heroku.

While it appears challenging, it might also show how to isolate the side-effects between the several microservices better.

Is this issue about requesting official Heroku support or about showing how to host on Heroku?

Else I would suggest to convert this issue into a discussion, or to move the conversation to the community forum https://forum.appflowy.io/
